jobs/async-upload/Makefile

```diff
@@ -48,9 +48,14 @@ test-with-cov:
 .PHONY: test-e2e
 test-e2e: deploy-latest-mr deploy-local-registry deploy-test-minio
 	@echo "Starting test-e2e"
-	-$(MAKE) test-e2e-run; STATUS=$$?
-	$(MAKE) test-e2e-cleanup
-	@exit $$STATUS
+	@set -e; \
```

Comment:
Does `set -e` do anything useful if we're catching the error with `||` and running both commands regardless of exit status? Did you just want to default to having it here for any future edits? (Same for `test-integration`)